src/main/webapp/js/courseSelector.js

```diff
@@ -98,15 +137,32 @@ export const CollegePlanner = (() => {
     return liElement;
   }
   window.addEventListener('load', () => {
-    getOptions();
+    getDepartmentOptions();
   });
   document.getElementById('add-selected').addEventListener('click', () => {
     addToSelected();
   });
+  document.getElementById('departments').addEventListener('change', () => {
+    getCourseOptions();
+  });
 
+  function getSelected() {
+    return selected;
+  }
+
+  function getCourses() {
+    return courses;
+  }
+
+  function getCourseInfo() {
+    return courseInfo;
+  }
 
   return {
-    selected: selected,
-    courses: courses,
+    getSelected: getSelected,
+    getCourses,
+    getCourses,
```

Comment:
What's going on here? you have two `getCourses`, and two `getCourseInfo`. If you're going to use shorthand properties in this dict, you should be consistent throughout too (change `getSelected`).